### ISSUE TITLE
weight decay from torchvisions blogpost

### ISSUE THREAD

**Issue by bw4sz** [weecology/DeepTreeAttention#129]
Just checking out a few of these improvements. 
https://pytorch.org/blog/how-to-train-state-of-the-art-models-using-torchvision-latest-primitives/
